docs/manual/manual_safety_systems.rst

```diff
@@ -180,8 +180,18 @@ Safety Parameters Handled by the Power Safety Client
 -------------------------------------------------------------
 The power safety client manages measurements regarding input power (voltage and current), internal reference power, motor current, microcontroller temperature, and coil temperature.
 
-The full listing of power safety parameters can be found in the :ref:`Power Safety Client table <power_safety>`. Each safety has a low end limit as well as a high end limit. 
-If the measured parameter drops below the low end or goes above the high end, a fault flag is raised, and can be viewed in the ``fault_now`` parameter. 
+The full listing of power safety parameters can be found in the :ref:`Power Safety Client table <power_safety_table>`. Each safety has a low end limit as well as a high end limit. 
+If the measured parameter drops below the low end or goes above the high end, a fault flag is raised, and can be viewed in the ``fault_now`` parameter. ``fault_now`` is 
+a bitmask used to show which flags are active, and is broken down as follows:
+
+* Bit 0: The microcontroller's Programmable Voltage Detector detected a voltage below acceptable limits
+* Bit 1: Power safety's input voltage reading has exceeded ``volt_input_high`` or has fallen below ``volt_input_low``
+* Bit 2: Power safety's reference voltage reading has exceeded ``vref_int_high`` or has fallen below ``vref_int_low``
+* Bit 3: Power safety's current input reading has exceeded ``current_input_high`` or has fallen below ``current_input_low``
```

Comment:
Might be good to clarify this is supply current, just to make it double obvious which type of current it is